modules/041-linstor/templates/linstor-node/linstorsatelliteset.yaml

```diff
@@ -51,7 +51,7 @@ metadata:
   {{- include "helm_lib_module_labels" (list . (dict "app" "linstor-node")) | nindent 2 }}
 spec:
   {{- include "helm_lib_priority_class" (tuple . "cluster-medium") | nindent 2 }}
-  {{- include "helm_lib_tolerations" (tuple . "wildcard") | nindent 2 }}
+  {{- include "helm_lib_tolerations" (tuple . "any-node") | nindent 2 }}
```

Comment:
Does `any-node` tolerates any node problems and planed cordon process?

If not, then it is wrong, because linstor-satelite should not be evicted with the normal pods, it should be able to handle all the requests from the linstor-controller (eg. to graceful shutdown or data eviction) after the user workload got evicted from the node.

Reply:
As can I see, it is not

https://github.com/deckhouse/lib-helm/blob/b2c73bba07bfccc68fb9aaad9977872ace5998b3/charts/helm_lib/templates/_node_affinity.tpl#L145-L162

Reply:
@nabokihms just clarified that these tolerations are added into every pod automatically by daemonset-controller:

```
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
  - effect: NoSchedule
    key: node.kubernetes.io/disk-pressure
    operator: Exists
  - effect: NoSchedule
    key: node.kubernetes.io/memory-pressure
    operator: Exists
  - effect: NoSchedule
    key: node.kubernetes.io/pid-pressure
    operator: Exists
  - effect: NoSchedule
    key: node.kubernetes.io/unschedulable
    operator: Exists
  - effect: NoSchedule
    key: node.kubernetes.io/network-unavailable
    operator: Exists
```

Thus we just need to add missing toleration for `storage-problems` and I'm fine with that